app/handlers/base.py

```diff
@@ -309,6 +310,8 @@ def error(self, message, data={}, status=400, extra={}):
         if not (status == 404 and self.request.method == "HEAD"):
             log(f'Error response returned by [{self.request.path}]: [{message}]')
 
+        if message.startswith("Access Error: Invalid"):
```

Comment:
Is this left over from the previous version?

Reply:
Ah yes, thanks,willupdate

Reply:
Updated